### DISCUSSION TITLE
Daily 13/10

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#130] [General]
Ayer terminé con el diagrama de colaboración de clases y empecé con el diagrama de clases, hoy terminamos con el diagrama de clases y preguntamos en el foro qué le parecía para que nos diga si hay algo que está muy mal. Hoy a la noche hay clase de SGC y de FS en otra aula, posiblemente esté en las dos pero también me ponga a hacer el diagrama relacional. No estoy seguro si los diagramas de caso de uso son para ahora o para después, la documentación del sprint dice para el sprint 2 pero en los foros menciona para el sprint 1. Mi expectativa es terminar con todo el viernes.

**Comment by NataliaAlvarezIspc** [comment]
Ayer hice el index, correcciones sobre los html subidos con anterioridad y corregiendo estilos en css. Subi todo al github.
Hoy participé en la reunion virtual de armado de diagrama de clases. Además estare dandole estilo al index,  corrigiendo los enlaces en todos los html y buscando tareas que esten pendientes para realizarlas. Por la tarde-noche participaré de las clases de SGC y FS.
Puede que por los tiempos no pueda llegar a hacer todo pero avanzaré lo más posible

**Comment by AhumadaCeleste** [comment]
Ayer comencé a ver los videos del profe Mainero y ejecute practica de Python. Me imprimí el material para comenzar a estudiar para el examen de proyecto. Hoy estoy corrigiendo lo que me indico Naty en el archivo de subasta. Sigo avanzando de a poco con lo que me queda de CSS. Mañana quisiera poder contribuir a completar si algo me quedara pendiente de hoy en el proyecto.

**Comment by Brian-Almada** [comment]
Ayer estuve estudiando boostrap.
Hoy por falta de tiempo sólo estoy viendo css del proyecto

**Comment by IngridAlcocer05** [comment]
Ayer subi el archivo html. Hoy repase css para afianzar conocimientos y poder finalizar con el diseño del catalogo en css.
